File: help/eng_gb/exercicios_da_ficha_10/Ficha10/Ficha10_002/Ficha10_002.php
<?php
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    for( $i = 1; $i < 21; $i = $i + 1 ) {
        if($i % 3 == 0 ){
            continue;
        }
        echo $i  .  " " ;
    }
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	quarta, 2022 out 12 19:32

?>

File: help/pt_pt/exercicios_da_ficha_10/Ficha10_002/Ficha10_002.cpp
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <iostream>
#include <cstring>
#include <cmath>
using namespace std;

// Class definition
class Ficha10_002 {

    public:
        int main() {
            int i  = 0 ;
            for( int i = 1; i < 21; i = i + 1 ) {
                if(i % 3 == 0 ){
                    continue;
                }
                cout << i  << " ";
            }
            return 0;
        }
};// end of class Ficha10_002
// Execute main function
int main(){
    Ficha10_002 application; // create object
    application.main(); // execute object
    return 0;
}

//C++  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 19:26
